internal/cluster/transition_test.go

```diff
@@ -3970,6 +3970,22 @@ var _ = Describe("Single node", func() {
 	})
 })
 
+var _ = Describe("ValidationResult sort", func() {
```

Comment:
Need to check the flow with `preprocess`

Reply:
you can do it here https://github.com/openshift/assisted-service/blob/master/internal/cluster/cluster_test.go#L2028